src/javascript/app/hooks/events.js

```diff
@@ -74,3 +74,7 @@ export const triggerPurchaseChange = () => triggerCustomEvent('purchaseChange');
 // Trade Change
 export const useTradeChange = () => useCustomEvent('tradeChange');
 export const triggerTradeChange = () => triggerCustomEvent('tradeChange');
+
+// Barrier Change
+export const useBarrierChange = () => useCustomEvent('barrierChange');
+export const triggerBarrierChange = () => triggerCustomEvent('barrierChange');
```

Comment:
Do we still need this? Can we leverage on the existing script where barrierChange is being updated and pass the values using `tradeManager`

Reply:
values are passed using tradeManager, but I separated the barrierChange to have the separate trigger